src/main/java/org/openpnp/machine/reference/driver/GcodeDriver.java

```diff
@@ -188,6 +187,33 @@ public void home(ReferenceHead head) throws Exception {
         for (ReferenceDriver driver : subDrivers) {
             driver.home(head);
         }
+
+        /*
+                The head camera for nozzle-1 should now be (if everythings homed correctly)
+                directly above the homing pin in the machine bed, use the head camera scan
+                for this and make sure this is exactly central - otherwise we move the camera
+                until it is, and then reset all the axis back to 0,0,0,0 as this is calibrated
+                home.
+         */
+        Part homePart=Configuration.get().getPart("FIDUCIAL-HOME");
+        if (homePart != null) {
+            Location tmp = new Location(LengthUnit.Millimeters,0.0,0.0,0.0,0.0);
```

Comment:
Let's make this location configurable in the XML. Just an @Element Location homingFiducialLocation would do the trick. Otherwise we're assuming that anyone who wants to use this will have their homing fiducial right under their endstops, and I think that situation will be rare.